docs/index.md

```diff
@@ -0,0 +1,102 @@
+#LORIS Neuroimaging Platform
+
+LORIS (Longitudinal Online Research and Imaging System) is a self-hosted web application that provides data- and project-management for neuroimaging research. LORIS makes it easy to manage large datasets including behavioural, clinical, neuroimaging and genetic data acquired over time or at different sites.
+
+A demo instance is available at https://demo.loris.ca.
+
+This Readme covers installation of the LORIS <b>v21</b> release on <b>Ubuntu</b>.
+([CentOS Readme also available](https://github.com/aces/Loris/blob/master/README.CentOS6.md)).
+
+Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/Setup) notes on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information.
+
+#### Heroku
+
+You can try LORIS on Heroku before installing it on your system. The project management and clinical data management functions of LORIS are available for experimenting with. Imaging functionality is not yet available.
+
+Deploy and log in with username *admin* and the password that's set up during deployment via ClearDB.
+
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/aces/Loris/tree/master)
+<br>
+
+## Installation
+
+#### System Requirements
+
+ * Apache **2.4** or higher
+ * MySQL >= 5.7 (or MariaDB >= 10.3) 
+ * PHP <b>7.2</b> or higher
+ * [Composer](https://getcomposer.org/)
+ * NodeJS <b>8.0</b> or higher
+
+These dependencies are subject to change so be sure to verify your version of MySQL and PHP when updating LORIS.
+
+### Install Steps
+
+Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information.
+
+1. Set up LINUX user lorisadmin and create LORIS base directory:
+
+    ```
+    sudo useradd -U -m -G sudo -s /bin/bash lorisadmin
+    sudo passwd lorisadmin
+    su - lorisadmin
+    ```
+
+    <b>Important: All steps from this point forward must be executed by lorisadmin user</b>
+
+    ```
+    sudo mkdir -m 775 -p /var/www/$projectname
+    sudo chown lorisadmin.lorisadmin /var/www/$projectname
+    ```
+
+    <i>$projectname ⇾ "loris" or one-word project name</i>
+
+2. Get code:
+    Download the latest release from the [releases page](https://github.com/aces/Loris/releases) and
+    extract it to `/var/www/$projectname`
+
+3. Run installer script to install core code, and libraries. The script will prompt for information and so that it can create directories automatically.
```

Comment:
My suggestion for removing a slight redundancy would be to modify:
"The script will prompt for information and so that it can create directories automatically."
and make it:
"The script will prompt for information and create directories automatically."

Reply:
Thanks @maltheism.  this comment should go on the Readme PR #5203

( cc @kongtiaowang )